project-icp/bee-pollinator-app
# Make Apiary Card Component

## Overview

Breaks down the Apiary Card into a number of components, and populates them via state. Also added some dummy state to populate the cards.

Connects #302 

### Demo

![image](https://user-images.githubusercontent.com/1430060/47813576-e296e800-dd21-11e8-98a8-5d6fc1d6bf67.png)

## Testing Instructions

* Check out this branch and `beekeepers build`
* Go to [:8000/?beekeepers](http://localhost:8000/?beekeepers) and see the cards
